### ISSUE TITLE
Tab Panels, Accordions, Cards and Screen Readers

### ISSUE THREAD

**Issue by mike-adams-edd** [Office-of-Digital-Innovation/California-State-Template-v5#89]
Hello, I have run into an issue where the accordions, tab panels, and cards are not being read by any of the screen readers that I am testing with (Jaws and NVDA). The header of the accordion will get read, and it reads the aria expanded attribute, but does not read the content in the expanded area. https://www.edd.ca.gov/payroll_taxes/FAQ_-_e-Services_for_Business.htm

For the tab panels and cards, it will not read any of the content inside of a tab panel. One way I did my own kind of work around was utilizing titles for each of the buttons in the cards so that way those with disabilities could still kind of navigate and find some information utilizing the keyboard and tabbing. https://www.edd.ca.gov/office_locator/

I have also tested the example pages on the template website http://template.webstandards.ca.gov/sample/accordions.html
http://template.webstandards.ca.gov/sample/tabs.html
and received the same results, the examples are not reading the content they should with the screen reader.

Thanks in advance!

Mike Adams

**Comment by kkoryaka** [comment]
Thank you for your inquiry. We will try to fix it in our next release.

**Comment by kkoryaka** [comment]
This issue should be fixed in our v5.013 and v5.5.10 beta releases.
